src/foam/u2/view/UnstyledTableView.js

```diff
@@ -278,10 +307,17 @@ foam.CLASS({
               }).
 
               // Render the table headers for the property columns.
-              forEach(columns_, function([axiomOrColumnName, overrides]) {
-                var column = typeof axiomOrColumnName === 'string'
-                  ? view.of.getAxiomByName(axiomOrColumnName)
-                  : axiomOrColumnName;
+              forEach(columns_, function([property, overrides]) {
```

Comment:
Should we use projection instead of retrieving value for a property for UnstyledTableView?